runtime/common/RuntimeMLIR.h

```diff
@@ -51,9 +51,9 @@ class Translation {
 
   /// Invoke the translation function with the given input and output streams.
   mlir::LogicalResult operator()(mlir::Operation *op,
-                                 llvm::raw_string_ostream &output,
-                                 bool printIR) const {
-    return function(op, output, printIR);
+                                 llvm::raw_string_ostream &output, bool printIR,
+                                 bool printIntermediateIR) const {
+    return function(op, output, printIR, printIntermediateIR);
```

Comment:
What is the intermediate IR and what is the IR?

Reply:
The pre-existing `printIR` refers to the final lowered LLVM IR that is sent to the QPU target, and the new `printIntermediateIR` refers to the per-pass MLIR outputs. I will rename `printIntermediateIR` to `printIntermediateMLIR` for clarity.

Reply:
Changed variable names in 96567e50 for clarity.